sphere-models/src/it/java/io/sphere/sdk/projects/queries/ProjectGetTest.java

```diff
@@ -15,6 +15,5 @@ public void execution() throws Exception {
         assertThat(project.getCountries()).overridingErrorMessage("countries").isNotEmpty();
         assertThat(project.getLanguages()).overridingErrorMessage("languages").isNotEmpty();
         assertThat(project.getCreatedAt()).overridingErrorMessage("createdAt").isNotNull();
-        assertThat(project.getTrialUntil()).overridingErrorMessage("trialUntil").isNotNull();
```

Comment:
because the project used on CI does not have this field (it is optional).
Or we could change the project on CI.